extensions/wikia/CreatePage/CreatePage.php

```diff
@@ -262,32 +262,38 @@ function wfCreatePageAjaxCheckTitle() {
 	if ( empty( $sTitle ) ) {
 		$result['result'] = 'error';
 		$result['msg'] = wfMsg( 'createpage-error-empty-title' );
+		$result['error'] = 'error-empty-title';
 	}
 	else {
 		$oTitle = Title::newFromText( $sTitle, $nameSpace );
 
 		if ( !( $oTitle instanceof Title ) || strpos( $sTitle, "#" ) !== false ) {
 			$result['result'] = 'error';
 			$result['msg'] = wfMsg( 'createpage-error-invalid-title' );
+			$result['error'] = 'error-invalid-title';
 		}
 		else {
 			if ( $oTitle->exists() ) {
 				$result['result'] = 'error';
 				$result['msg'] = wfMsg( 'createpage-error-article-exists', array( $oTitle->getFullUrl(), $oTitle->getText() ) );
+				$result['error'] = 'error-article-exists';
 			}
 			else { // title not exists
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)